webpack.config.js

```diff
@@ -19,6 +20,10 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
         publicDir = composerConfig.extra['public-dir'];
     }
 
+    const composerLock = JSON.parse(fs.readFileSync(path.resolve(projectRootPath, 'composer.lock')));
+    const suluPackage = composerLock.packages.find((p) => p.name === 'sulu/sulu');
+    const suluVersion = suluPackage ? suluPackage.version : 'unknown';
```

Comment:
Wouldn't it make more sense to return `undefined` in case the package version is not known? Then it would be easier to handle that case differently, in case that is necessary at some point.

Reply:
have not thought about that - `undefined` sounds good too